packages/App/src/Registaration/LogIn.tsx

```diff
@@ -0,0 +1,215 @@
+import React, {FC, useState, useContext} from 'react';
+import {
+  View,
+  StyleSheet,
+  TextInput,
+  ImageBackground,
+  Text,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+// import AsyncStorage from '@react-native-async-storage/async-storage';
+import BG from '../../Pics/Group32.png';
+import passwordUnvisible from '../../Pics/passwordUnvisible.png';
+import passwordVisible from '../../Pics/passwordVisible.png';
+import ButtonApp from '../components/ButtonApp';
+import {Context} from '../../App';
+import {Formik} from 'formik';
+import * as yup from 'yup';
+
+const logInSchema = yup.object({
+  email: yup.string().required().email(),
+  password: yup.string().required(),
```

Comment:
Maybe we should add a stricter validation for password. Let's say it has to be at least 6 digits long